src/glue/operations/init_concept_count.py

```diff
@@ -24,6 +25,9 @@ def run(config: GlueConfig, api: WebAPIClient):
     with MultiDB(*config.cdm_db_params()) as cdm_db:
         logger.info("starting")
         ddl = api.get_achilles_ddl()
+        # remove c-style comments which may currently contain strings that look
+        # like named param references
+        ddl = re.sub(r"/\*.+?\*/", "", ddl, flags=re.DOTALL)
```

Comment:
I don't love this because it seems a little error prone. It's a lot of responsibility to give the the non-greedy modifier. But if it gets the job done for sql server I'm willing to go with it.